`textures/Texture3D.js` seems to be missing - I'm getting a build error:
```
$ npm run-script build

> three@0.94.0 build /tmp/tmp.jm7GRQ5b8Z/three.js-mirror
> rollup -c


src/Three.js → build/three.js, build/three.module.js...
[!] Error: Could not resolve './textures/Texture3D.js' from src/Three.js
Error: Could not resolve './textures/Texture3D.js' from src/Three.js
    at error (/tmp/tmp.jm7GRQ5b8Z/three.js-mirror/node_modules/rollup/dist/rollup.js:199:15)
    at /tmp/tmp.jm7GRQ5b8Z/three.js-mirror/node_modules/rollup/dist/rollup.js:20784:25
    at <anonymous>
    at process._tickCallback (internal/process/next_tick.js:188:7)
    at Function.Module.runMain (module.js:695:11)
    at startup (bootstrap_node.js:191:16)
    at bootstrap_node.js:612:3

npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! three@0.94.0 build: `rollup -c`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the three@0.94.0 build script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     ~/.npm/_logs/2018-07-23T14_47_03_378Z-debug.log
```

--- a/src/renderers/WebGLRenderer.js
+++ b/src/renderers/WebGLRenderer.js
@@ -2357,6 +2357,7 @@ function WebGLRenderer( parameters ) {
 
 		// backwards compatibility: peel texture.texture
 		return function setTexture3D( texture, slot ) {
+
 			textures.setTexture3D( texture, slot );
 
 		};

--- a/src/textures/Texture3D.js
+++ b/src/textures/Texture3D.js
@@ -1,0 +1,26 @@
+/**
+ * @author Artur Trzesiok
+ */
+
+import {Texture} from './Texture.js';
+import {NearestFilter} from '../constants.js';
+
+function Texture3D(data, width, height, length, format, type, mapping, wrapS, wrapT, magFilter, minFilter, anisotropy, encoding) {
+
+    Texture.call(this, null, mapping, wrapS, wrapT, magFilter, minFilter, format, type, anisotropy, encoding);
+
+    this.image = {data: data, width: width, height: height, length: length};
+
+    this.magFilter = magFilter !== undefined ? magFilter : NearestFilter;
+    this.minFilter = minFilter !== undefined ? minFilter : NearestFilter;
+
+    this.generateMipmaps = false;
+    this.flipY = false;
+
+}
+
+Texture3D.prototype = Object.create(Texture.prototype);
+Texture3D.prototype.constructor = Texture3D;
+Texture3D.prototype.isTexture3D = true;
+
+export {Texture3D};

